### PR TITLE
[16.0][IMP] excel_import_export: possibility to apply unidecode

### PR DESCRIPTION
**Pull request by RodrigoBM**
Add the possibility to apply unidecode to the data to be set in the exporter.
It is possible that the file you export has only the characters ascii.

e.g.
unidecode.unidecode("ESPAÑA")
'ESPANA'
unidecode.unidecode("forçut")
'forcut'

It is possible that the system where you import the file only supports ascii characters.

### CODE REVIEW

**Review by xAdrianCif — APPROVED** [review]
Tech review ok! thanks!

**Review by liebana — APPROVED** [review]
Functional review OK

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Adding extra dependencies in a merged module is not advisable, and in this case being an extra feature, it should be done in a new module `excel_import_export_unidecode` or similar.

**Comment by RodrigoBM** [comment]
Hi @pedrobaeza,

I understand what you are saying, but I don't see a way to do this in another module, the methods are not very inheritable.

Best regards

**Comment by pedrobaeza** [comment]
You can provide hooks in this base module for easing that.

**Comment by RodrigoBM** [comment]
> You can provide hooks in this base module for easing that.

Done @pedrobaeza

File: requirements.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ openpyxl
 openupgradelib
 pygount
 sentry_sdk<=1.9.0
+unidecode
```
Comment (pedrobaeza) [inline requirements.txt:10]: Put this in the other commit.

**Review by pedrobaeza — COMMENTED** [review]
The rest seems OK.

File: requirements.txt (hunk 1 above)
Comment (RodrigoBM) [reply requirements.txt:10]: Done, I forgot this in the other commit.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**